tensorflow/python/tfe_wrapper.cc

```diff
@@ -524,68 +539,64 @@ PYBIND11_MODULE(_pywrap_tfe, m) {
         TFE_Py_RegisterFallbackExceptionClass(e.ptr()));
   });
 
-  m.def(
-      "TFE_GetMemoryInfo", [](py::handle& ctx, const char* device_name) {
-        auto* context =
-            reinterpret_cast<tensorflow::ImmediateExecutionContext*>(
-                tensorflow::InputTFE_Context(ctx));
-
-        tensorflow::DeviceNameUtils::ParsedName input_device_name;
-        if (!tensorflow::DeviceNameUtils::ParseFullOrLocalName(
-                device_name, &input_device_name)) {
-          tensorflow::ThrowValueError(
-              absl::StrFormat("Failed parsing device name: '%s'", device_name)
-                  .c_str());
-        }
+  m.def("TFE_GetMemoryInfo", [](py::handle& ctx, const char* device_name) {
```

Comment:
Why is TFE_GetMemoryInfo changing? Is this just a merge conflict?

Reply:
It is modified by clang-format